suzieq/engines/pandas/path.py

```diff
@@ -853,7 +853,15 @@ def get(self, **kwargs) -> pd.DataFrame:
                         vrfchk = dest_device_iifs[destdevkey]["vrf"]
                         rev_df = self._rpf_df.query(
                             f'hostname == "{device}" and vrf == "{vrfchk}"')
-                        if rev_df.empty:
+                        if rev_df.empty and is_l2:
+                            # for the case if this node is the ingress VTEP
+                            rev_df = self._arpnd_df.query(
+                                f'hostname == "{device}" and '
+                                f'ipAddress == "{dest}"')
+                            if rev_df.empty and is_l2:
```

Comment:
If we entered here, `is_l2` is already True. We can simplify the check as follows
```suggestion
                            if rev_df.empty:
```